### DISCUSSION TITLE
WPF的textbox如何设置为密码框样式

### DISCUSSION THREAD

**Discussion by jlennonxzy** [BYJRK/DotNet-Discussions#98] [MVVM]
显示黑点或者星号。不能用passwordBox，因为我要绑定这个内容，所以必须用依赖属性。谢谢指教。

**Comment by BYJRK** [comment]
参考一下这个视频，给 PasswordBox 添加自定义附加属性

https://www.youtube.com/watch?v=trR5RQGEteM

**Reply by jlennonxzy** [reply]
谢谢指导。
